app/models/bishop.rb

```diff
@@ -1,13 +1,17 @@
-# This is code for how the Bishop should move
+legal# This is code for how the Bishop should move
```

Comment:
Looks like I started typing for something else and missed it...great job me!  Way to go!  lol